CodeEdit/Features/SourceControl/Views/SourceControlPullView.swift

```diff
@@ -35,6 +39,14 @@ struct SourceControlPullView: View {
             .formStyle(.grouped)
             .scrollDisabled(true)
             .scrollContentBackground(.hidden)
+            .onAppear {
```

Comment:
Is this a similar situation as the Settings view where we can't use `.task`?

Reply:
I believe so